earth_enterprise/Upgrade.md

```diff
@@ -10,7 +10,7 @@ sudo ./install_fusion.sh
 sudo ./install_server.sh
 ```
 
-Alternatively, instead of using scripts, on Redhat and Centos, you can upgrade GEE 5.2.0 to GEE 5.2.1 using [RPM packages] (https://github.com/google/earthenterprise/blob/master/earth_enterprise/BUILD_RPMS.md)
+Alternatively, instead of using scripts, on Redhat and Centos, you can upgrade GEE 5.2.0 to GEE 5.2.1 using [RPM packages](https://github.com/google/earthenterprise/blob/master/earth_enterprise/BUILD_RPMS.md)
 
 1. If you decide that you want to uninstall GEE 5.2.0 before installing GEE 5.2.1, first make sure to backup your PostgreSQL databases. Please keep in mind that the database backup, made by 5.2.0, would not be compatible with GEE 5.2.1 PostgreSQL databases.
 
```

Comment:
Typo: There seems to be an unintentional `"` at the beginning of the command on line 20 below.

Reply:
Fixed.